src/game/objects/general/bridge.c

```diff
@@ -223,3 +252,11 @@ void Bridge_Tilt2Ceiling(
         *height = level + STEP_L;
     }
 }
+
+void Bridge_Initialise(int16_t item_num)
+{
+    // Some bridges at floor level are embedded into the floor.
+    // This checks if bridges are below a room's floor level
+    // and moves them up.
+    Bridge_FixEmbeddedPosition(item_num);
+}
```

Comment:
Nitpick: let's put it in an order that matches the header file (e.g. after setup draw bridge function)

Reply:
I am NOT (!) suggesting we do anything since it's all cosmetic, but I did notice a lot of the .h files do not follow the function order in the .c file. I agree it should match lol. Maybe some time (far) into the future lol.

Reply:
I agree these need fixing. We'll get there, at some point. Looking forward to it :D